Lib/idlelib/config-extensions.def

```diff
@@ -32,68 +52,14 @@
 # See config-keys.def for notes on specifying keys and extend.txt for
 # information on creating IDLE extensions.
 
-[AutoComplete]
-enable=True
-popupwait=2000
-[AutoComplete_cfgBindings]
-force-open-completions=<Control-Key-space>
-[AutoComplete_bindings]
-autocomplete=<Key-Tab>
-try-open-completions=<KeyRelease-period> <KeyRelease-slash> <KeyRelease-backslash>
```

Comment:
Here you accomplished effect 1 above, `event_add(...)`, by adding the event-sequences pair to core bindings.  But this has the bug of adding then to the configure Keys tab.  Core bindings are configurable bindings.  I will add `event_add` elsewhere when I decide where.